app/models/pipeline_run.rb

```diff
@@ -82,7 +82,10 @@ class PipelineRun < ApplicationRecord
                         "taxon_byteranges" => "db_load_byteranges" }.freeze
   # Note: reads_before_priceseqfilter, reads_after_priceseqfilter, reads_after_cdhitdup
   #       are the only "job_stats" we actually need for web display.
-  REPORT_READY_OUTPUT = "taxon_counts".freeze
+  HOST_FILTERING_OUTPUT = "ercc_counts".freeze
+  ALIGNMENT_OUTPUT = "taxon_counts".freeze
```

Comment:
nit: the philosophy of result_monitor / output_states is that they are in principle decoupled from the run_stages / pipeline_monitor. So you could have multiple outputs per stage in the future. So more appropriate names for the constants would be: ERCC_OUTPUT, TAXON_COUNT_OUTPUT, TAXON_BYTERANGE_OUTPUT. But otherwise looks more or less good to go